simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -158,7 +167,7 @@
 
       <sensor name="sick_lidar" type="ray">
         <pose>0.5334 -0.0254 -0.6858 0 0 0</pose>
-        <visualize>false</visualize>
+        <visualize>true</visualize>
```

Comment:
This was intentionally disabled because it makes working in the GUI unnecessarily irritating and serves no real purpose.